testCommon/src/testFixtures/resources/common.sh

```diff
@@ -37,7 +37,7 @@ function create_repo() {
   cd $dir
   shift
   git init "$@"
-  if [ "$(expr substr $(uname -s) 1 10)" != "MINGW64_NT" && "$(expr substr $(uname -s) 1 10)" != "MINGW32_NT" ]; then
```

Comment:
This was the culprit... the expression always evaluated to false, coz `expr substr` is some weird stuff that doesn't always work, see [SC2308](https://github.com/koalaman/shellcheck/wiki/SC2308#substr)